packages/2019-template/src/state/request-adapter.js

```diff
@@ -0,0 +1,4 @@
+import fetch from "cross-fetch";
+import { makeFetchAdapter } from "reduxful";
+
+export default makeFetchAdapter(fetch);
```

Comment:
I noticed this as well. I'm agreed on adding the `request-adapter.js` file if it doesn't already exist.

I was thinking we could just inject the dependencies using the same hygen template techniques from this PR. Generally speaking, I don't like adding dead code, which is why I prefer injecting if not present rather than assuming that a new package will be making cards... but then again only the disaster game so far has not included cards. I'll add this to my final summary.